core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/variableshadowing/VariableShadowingUnitTest.kt

```diff
@@ -0,0 +1,207 @@
+@file:Suppress("UNUSED_VARIABLE", "UNUSED_ANONYMOUS_PARAMETER")
+
+package com.baeldung.variableshadowing
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class VariableShadowingUnitTest {
+
+    @Test
+    fun `top-level variable shadowing`(){
```

Comment:
We still have formatting issues like this. I checked out the PR and applied the Baeldung formatter and it fixed them:
https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/intellij
```suggestion
    fun `top-level variable shadowing`() {
```